### PR TITLE
<docs>(readme): add runbook for registration team

### PR DESCRIPTION
**Pull request by david30907d**

### CODE REVIEW

**Comment by david30907d** [comment]
cc @jordanSu

File: README.md [hunk 1]
```diff
@@ -86,7 +88,8 @@ xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx,Speaker 講者票（with Pyckage）
 
 #### Where to Put These Tokens
 
-Tokens should be stored under "tokens" folder.
+1. `gcloud beta compute ssh --zone "asia-east1-b" "pycontw-website" --project "pycontw-225217"`
+2. `cd /home/jordan/pycon-regbot`
```
Comment (jordanSu) [inline README.md]:
```suggestion
1. Go to the prod server we deploy pycon-regbot
`$ gcloud beta compute ssh --zone "asia-east1-b" "pycontw-website" --project "pycontw-225217"`
2. Change directory to `/home/jordan/pycon-regbot`, and you should put token files here.
`$ cd /home/jordan/pycon-regbot`

```

**Review by jordanSu — COMMENTED** [review]
@david30907d I made some change, let me know if this explains better.

**Review by jordanSu — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**